opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/SpanShimTest.java

```diff
@@ -104,6 +109,48 @@ void finish_micros() {
     assertThat(spanData.getEndEpochNanos()).isEqualTo(micros * 1000L);
   }
 
+  @Test
```

Comment:
The code has a ton of if statements so I suspect just these tests isn't covering it enough :) Didn't do a full look, but for example, if an event has both exception and error fields. Please flesh out all corner cases.

Reply:
yeah, looks like we don't have any coverage for anything but string values at the moment. Would be really good to add some coverage for numeric and boolean keys as well.

Reply:
added